### PR TITLE
(MAINT) Improve status proxy docs

### PR DESCRIPTION
**Pull request by jpinsonault**
Follow up to [PR 22](https://github.com/puppetlabs/trapperkeeper-status/pull/22) to fixup the status proxy service docs

@camlow325 Thanks for the good feedback, I think I've addressed it all
ping @cprice404 for review if you have any input

### CODE REVIEW

**Comment by MSLilah** [comment]
:+1:

File: documentation/status-proxy-service.md [hunk 1]
```diff
@@ -114,10 +117,11 @@ The important things to note are:
 * The status proxy service and the status service are running on separate
   webservers and ports
 * There is a new section in the config, `status-proxy`, with:
-  * a url pointing the proxy to the status service
+  * A url pointing the proxy to the status service. Note that he hostname of
```
Comment (camlow325) [inline documentation/status-proxy-service.md]: s/he/the

**Comment by camlow325** [comment]
:+1: other than the one small typo

**Comment by jpinsonault** [comment]
@camlow325 Fixed the typo

**Comment by camlow325** [comment]
:shipit:

**Pull request merged**